docker/prowbazel/Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = istio-testing
-VERSION ?= 0.5.12
+VERSION ?= 0.5.13
```

Comment:
Same for the other references.
```suggestion
VERSION ?= 0.5.14
```

Reply:
Ok, I did not see 0.5.13 in the README file so I used it and I think I've already pushed the image with that tag :). Anyway I updated everything to 0.5.14 now.

Reply:
1) Prow pushes the image on `postsubmit`. I don't think it is good practice to manually push images overwriting ones that already exist.
2) That is my fault for not updating the README, I am retroactively doing it here #2011. The version was clearly defined in `Makefile` if you were to rebase.
3) I am not sure what tags you pushed, but now `0.5.12`, `0.5.13`, and `0.5.14` should all be separate images. As long as the `0.5.14`  you pushed has my changes from #2001 it should be fine if nothing uses those old images.
![image](https://user-images.githubusercontent.com/9957358/67611288-72ad1900-f74d-11e9-93c3-4915399ff7a3.png)

Reply:
Oh sorry I don't realize there is a postsubmit that does image push. I thought it was always done manually. I won't push manually, thanks!

Yeah, it should be fine for this updates, only 0.5.14 should be used now and let's merge this PR to make sure the tag has all changes?

Reply:
Can we change the postsubmit to avoid overwriting existing images?

Reply:
#2014

Reply:
No problem @bianpengyuan . I was more worried about the image overwriting than anything else. Can you rebase, then we can get this merged and updated?

Reply:
Thanks! Rebased.